xml_inscriptions/transcribed/MA.Bos.MFA.G.35.61.xml

```diff
@@ -0,0 +1,169 @@
+                  <origin>
+                     <date evidence="#unspecified" notBefore="-0526" notAfter="-0499">about 525–500
+                        B.C.</date>
+                     <placeName ref="europe.greece">Galaxidi/ancient Haleion</placeName>
+                  </origin>
+                  <provenance type="transferred" subtype="purchased">
+                     <p> in <date when="1935">1935</date>, sold by <persName role="seller">Theodoros Zompoulakis</persName>, of <placeName>Athens</placeName>, to the <orgName role="buyer">MFA</orgName> for $80. (Accession Date: <date when="1935">March 7, 1935</date>)</p>
+                  </provenance>
+                  <provenance type="observed" subtype="reported">
+                     <p> said to be from <placeName>Galaxidi</placeName>, Greece (ancient <placeName>Haleion</placeName>placeName>).</p>
+                  </provenance>
+                  
+                  
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CVB"/>
+                  <biblScope unit="item">no. 681</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#LSAG"/>
+                  <biblScope unit="item">108, no. 17</biblScope>
+               </bibl>
+               <bibl>
+                  <ref type="external"
+                     target="https://collections.mfa.org/objects/153322/miniature-chariot-wheel?ctx=07d8ceab-7151-48a3-8aec-f0488f063b84&amp;idx=0"
+                     >MFA web entry</ref>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+            href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-07-13" who="Jean Wanlass">added metadata, transcription,
+            translation</change>
+         <change when="2023-07-25" who="Jean Wanlass">minor edits/improvements</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.35.61.jpg"/>
+      </surface>
+   </facsimile>
+   <text>
+      <body>
+         <div type="edition">
+            <ab><lb n="1"/>Φαλὰς Πεδιαρχεῖον ἀνέθηκε τοπόλονι </ab></div>
```

Comment:
I don't agree with the reading/translation here (which seems to follow the MFA website); I haven't really been able to find parallels and Πεδιαρχεῖον can't modify Φάλας as a patronymic. I've revised to fit my interpretation of what's going on (following Bousquet's and Robert's comments), but @jbodel might want to take a look.